conjure/serverwriter.go

```diff
@@ -354,21 +352,28 @@ func astForDecodeHTTPParamInternal(methodBody *jen.Group, argName string, argTyp
 				g.Id(outVarName).Op("=").Append(jen.Id(outVarName), jen.Id("convertedVal"))
 			})
 		}
+	case *types.Set:
```

Comment:
Can you add a unit test for sets in `serverwriter_test.go` since it looks like we don't have any yet.

Reply:
Yup, done!